### PR TITLE
resume chunked uploads

### PR DESCRIPTION
**Pull request by tobiasKaminsky**

### CODE REVIEW

File: src/com/owncloud/android/operations/UploadFileOperation.java [hunk 1]
```diff
@@ -363,7 +363,7 @@ protected RemoteOperationResult run(OwnCloudClient client) {
                     (new File(mFile.getStoragePath())).length() >
                             ChunkedUploadRemoteFileOperation.CHUNK_SIZE ) {
                 mUploadOperation = new ChunkedUploadRemoteFileOperation(mFile.getStoragePath(),
-                        mFile.getRemotePath(), mFile.getMimetype(), mFile.getEtagInConflict());
+                        mFile.getRemotePath(), mFile.getMimetype(), mFile.getEtagInConflict(), mContext);
```
Comment (AndyScherzinger) [inline src/com/owncloud/android/operations/UploadFileOperation.java]: as mentioned in the lib PR, I would make the context the first parameter.

File: src/com/owncloud/android/MainApp.java [hunk 2]
```diff
@@ -79,7 +80,8 @@ public void onCreate(){
             // Set folder for store logs
             Log_OC.setLogDataFolder(dataFolder);
 
-            Log_OC.startLogging();
+            //TODO: to be changed/fixed whenever SD card support gets merged.
+            Log_OC.startLogging(Environment.getExternalStorageDirectory().getAbsolutePath());
```
Comment (tobiasKaminsky) [inline src/com/owncloud/android/MainApp.java]: What path is this where you are logging?
This must be the directory where the base directory of the application is.

Comment (AndyScherzinger) [inline src/com/owncloud/android/MainApp.java]: Need to further investigate this one. Just got the code back to compile.

Comment (AndyScherzinger) [inline src/com/owncloud/android/MainApp.java]: @tobiasKaminsky see the corresponding change of @przybylski in the lib project: https://github.com/nextcloud/android-library/pull/5/files#diff-0eb21eac7b9b2e2b07a7f7a7945ce689L76 
So this simply sets the path as it has been before.

**Comment by AndyScherzinger** [comment]
Code looks good: :+1: 

Didn't have the time to test it yet though.

**Comment by AndyScherzinger** [comment]
@tobiasKaminsky this has been code reviewed and has been out in the wild for some time, right? Merge? :grin: (just asking :wink:)

**Comment by tobiasKaminsky** [comment]
Merge :)

**Pull request merged**